### PR TITLE
Add adapter for Axum -> Vercel

### PR DESCRIPTION
**Pull request by dglsparsons**

### CODE REVIEW

File: crates/vercel_axum/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[dependencies]
+axum = "0.7"
+base64 = "0.22"
+http-body-util = "0.1"
+tower = "0.4"
+tower-service = "0.3"
+serde_json = "1.0"
+# vercel_runtime = "1.1.3"
+vercel_runtime = { version = "1.1.3", path = "../vercel_runtime" }
```
Comment (ecklf) [inline crates/vercel_axum/Cargo.toml]:
```suggestion
vercel_runtime = "1.1.3"
# vercel_runtime = { version = "1.1.3", path = "../vercel_runtime" }
```

File: crates/vercel_axum/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[package]
+name = "vercel_axum"
+version = "1.1.2"
```
Comment (ecklf) [inline crates/vercel_axum/Cargo.toml]:
```suggestion
version = "1.1.3"
```

File: crates/vercel_runtime/src/response.rs
Comment (ecklf) [inline crates/vercel_runtime/src/response.rs:1]: Is this a required change for this to work or just a simplification (I assume the latter)?

**Comment by ecklf** [comment]
It would be nice if the README had instructions in how to use this

File: crates/vercel_runtime/src/response.rs
Comment (dglsparsons) [reply crates/vercel_runtime/src/response.rs:1]: The custom `serialize_headers` implementation is a bit buggy. Specifically, it doesn't handle headers that repeat, which is often the case for `set-cookie`. So it's kinda simultaneously a bug fix and a simplification.

**Review by ecklf — APPROVED** [review]

**Pull request merged**